src/validation/validators/nameValidator.ts

```diff
@@ -1,12 +1,12 @@
 export const nameValidator = (
-  column: string,
+  displayName: string,
   value: unknown
 ): { message: string } | null => {
   if (!value) return null;
   if (typeof value !== "string") return null;
 
   if (!/^[^@]+$/.test(value)) {
-    return { message: `${column}は記号は入力できません。` };
+    return { message: `${displayName}は記号は入力できません。` };
```

Comment:
```suggestion
    return { message: `${displayName}に記号は入力できません。` };
```
変更ついでって感じですが, こちらのほうが自然かも 🤔

Reply:
たしかに